### PR TITLE
Update VarTierDiseaseDBFalse.R

### PR DESCRIPTION
**Pull request by hyunhwan-bcm**
`VarTierDiseaseDBFalse.R` is the bottleneck for the current pipeline. This PR contains the updated  `VarTierDiseaseDBFalse.R`  from the current `nextflow_branch`, this will help to resolve the processing delay issue.

### CODE REVIEW

**Comment by hyunhwan-bcm** [comment]
@jylee-bcm can you test this on both hg19/hg38? Thank you, we may need to use a small input for it.

File: run/VarTierDiseaseDBFalse.R [hunk 1]
```diff
@@ -1,381 +1,352 @@
...
-args = commandArgs(trailingOnly=TRUE)
+args <- commandArgs(trailingOnly = TRUE)
 refg <- args[2]
-# load database: OMIM inheritance ####
-# genemap2.Inh.F <- readRDS(file.path("/run/genemap2.Inh.F.rds"))
-genemap2.Inh.F <- read.delim2(file.path(paste0('/run/data_dependencies/var_tier/',refg,'/genemap2.Inh.F.txt')))
 
-# set the input output file / directory ####
+cat("STEP0. load database: OMIM inheritance ####\n")
+omim_inheritance <- file.path(paste0("var_tier/", refg, "/genemap2.Inh.F.txt"))
```
Comment (jylee-bcm) [inline run/VarTierDiseaseDBFalse.R]: `paste0('/run/data_dependencies/var_tier/',refg,'/genemap2.Inh.F.txt')`

File: run/VarTierDiseaseDBFalse.R [hunk 2]
```diff
@@ -1,381 +1,352 @@
...
-# in.f <- paste0(args[1],"_scores.csv")
 in.f <- paste0(args[1],"_scores.csv")
 out.f <- paste0(args[1],"_Tier.v2.tsv")
 
-# load libraries ####
-library("data.table")
-library("dplyr")
-
-
+genemap2.Inh.F <- readr::read_tsv(omim_inheritance)
```
Comment (jylee-bcm) [inline run/VarTierDiseaseDBFalse.R:25]: use path()

File: run/VarTierDiseaseDBFalse.R [hunk 3]
```diff
@@ -1,381 +1,352 @@
...
-
-
+VEP.Tier.wInh$AD.matched <- ifelse(VEP.Tier.wInh$TierAD <= 2 &
+                                     (VEP.Tier.wInh$dominant == 1), 1, 0)
+VEP.Tier.wInh$AR.matched <- ifelse(VEP.Tier.wInh$TierAR <= 2 &
+                                     (VEP.Tier.wInh$recessive == 1), 1, 0)
+
+write.table(
+  VEP.Tier.wInh,
+  out.f,
```
Comment (jylee-bcm) [inline run/VarTierDiseaseDBFalse.R]: use path()

**Review by jylee-bcm — CHANGES_REQUESTED** [review]
I will update the pr soon :D

**Review by jylee-bcm — APPROVED** [review]
Now it's working, but I could not perform the checking on outputs yet.

**Comment by hyunhwan-bcm** [comment]
will be merged with `nextflow_conversion`

**Pull request closed without merging**